### PR TITLE
[exporter/logging] log min and max for histograms

### LINKED ISSUE #5518: [exporter/logging] Histogram `min` and `max` fields are not logged.
**Is your feature request related to a problem? Please describe.**
The metrics `logging` exporter currently does not log `min` and `max` fields in Histograms in Histograms and Exponential Histograms.

**Describe the solution you'd like**
The `logging` exporter should also log `min` and `max` fields in Histograms and Exponential Histograms.

### PR DESCRIPTION
**Pull request by pichlermarc**
**Description:** 
The `logging` exporter did not show `min` and `max` of histograms. This PR adds logging for `min` and `max` on both Histograms and Exponential Histograms.

**Link to tracking Issue:** Fixes #5518

**Testing:** 
- Existing unit tests
- Manual testing with `min` and `max` fields on histograms during implementation of https://github.com/open-telemetry/opentelemetry-js/pull/3032

### CODE REVIEW

File: internal/otlptext/databuffer.go [hunk 1]
```diff
@@ -118,6 +118,8 @@ func (b *dataBuffer) logDoubleHistogramDataPoints(ps pmetric.HistogramDataPointS
 		b.logEntry("Timestamp: %s", p.Timestamp())
 		b.logEntry("Count: %d", p.Count())
 		b.logEntry("Sum: %f", p.Sum())
+		b.logEntry("Min: %f", p.Min())
+		b.logEntry("Max: %f", p.Max())
```
[the remark this answers is not in the document]
Comment (pichlermarc) [reply internal/otlptext/databuffer.go]: Addressed in [66405f1](https://github.com/open-telemetry/opentelemetry-collector/pull/5520/commits/66405f1a2e7c8073c6e4f1ba0b5d9e8f69f99fef). Thanks for the feedback! :slightly_smiling_face:
I also added a check for `sum` as it is optional too in OTLP v0.17, and added `min` and `max` to the test data in [4eb70cd](https://github.com/open-telemetry/opentelemetry-collector/pull/5520/commits/4eb70cd994d52e9be42b15eb129ebfb96da93191).

**Comment by pichlermarc** [comment]
go 1.18 unit tests are currently failing. I'm investigating.

File: internal/otlptext/databuffer.go [hunk 2]
```diff
@@ -146,6 +157,14 @@ func (b *dataBuffer) logExponentialHistogramDataPoints(ps pmetric.ExponentialHis
 		b.logEntry("Count: %d", p.Count())
 		b.logEntry("Sum: %f", p.Sum())
```
[the remark this answers is not in the document]
Comment (codeboten) [reply internal/otlptext/databuffer.go:158]: Now that https://github.com/open-telemetry/opentelemetry-collector/pull/5530 merged, please update the PR to check `HasSum` here as well

[the remark this answers is not in the document]
Comment (pichlermarc) [reply internal/otlptext/databuffer.go]: Done in [2883283](https://github.com/open-telemetry/opentelemetry-collector/pull/5520/commits/2883283543b586b5604071ea6159230affffa6a7). Thanks for updating the proto version! :slightly_smiling_face:

**Review by codeboten — APPROVED** [review]
Thanks for fixing this and following up after my change!

**Pull request merged**